ethcore/evm/src/interpreter/gasometer.rs

```diff
@@ -376,7 +376,7 @@ fn calculate_eip1283_sstore_gas<Gas: evm::CostType>(schedule: &Schedule, origina
 	Gas::from(
 		if current == new {
 			// 1. If current value equals new value (this is a no-op), 200 gas is deducted.
```

Comment:
This comment is now outdated, can be 200 or 800 (if eip2200_advance_transition is activated)